<launch>
	<!-- Start Reynolds controller for each Sphero -->
	<group ns="sphero_0">
		<node pkg="sphero_formation" type="reynolds_controller.py" name="reynolds">
			<param name="init_vel_x" type="double" value="0"/>
			<param name="init_vel_y" type="double" value="0.5"/>
		</node>
	</group>

	<group ns="sphero_1">
		<node pkg="sphero_formation" type="reynolds_controller.py" name="reynolds">
			<param name="init_vel_x" type="double" value="0"/>
			<param name="init_vel_y" type="double" value="0.5"/>
		</node>
	</group>

	<group ns="sphero_2">
		<node pkg="sphero_formation" type="reynolds_controller.py" name="reynolds">
			<param name="init_vel_x" type="double" value="0"/>
			<param name="init_vel_y" type="double" value="0.5"/>
		</node>
	</group>

	<group ns="sphero_3">
		<node pkg="sphero_formation" type="reynolds_controller.py" name="reynolds">
			<param name="init_vel_x" type="double" value="0"/>
			<param name="init_vel_y" type="double" value="0.5"/>
		</node>
	</group>
</launch>